packages/domain-picker/src/components/index.tsx

```diff
@@ -119,6 +119,9 @@ export interface Props {
 
 	/** Vendor string for domain suggestions */
 	vendor?: string;
+
+	/** Hides the recommendation label */
+	hideRecommendationLabel?: boolean;
```

Comment:
The affirmative form of the props is generally easier to read. You may go with this form which would be also consistent with `showSearchField` and `showDomainCategories` props.
```suggestion
	/** Whether to show the recommendation label or not. Defaults to true */
	showRecommendationLabel?: boolean;
```

Reply:
Thanks @razvanpapadopol  that is indeed more readable :)
It's something that I do follow but since the default behaviour was to show the label I went with the negative form of the prop!